src/main/java/net/sf/jabref/logic/groups/AllEntriesGroup.java

```diff
@@ -32,22 +32,14 @@ public AllEntriesGroup() {
         super(Localization.lang("All entries"), GroupHierarchyType.INDEPENDENT);
     }
 
-    public static AbstractGroup fromString(String s, int version) throws Exception {
+    public static AbstractGroup fromString(String s) throws Exception {
         if (!s.startsWith(AllEntriesGroup.ID)) {
             throw new Exception(
```

Comment:
In reference to my above comment: We could certainly throw a better type of Exception here. What about an `IllegalArgumentException`?